frame.js

```diff
@@ -0,0 +1,74 @@
+class Frame {
+  constructor() {
+    this.rollCounter = 1
+    this.rolls = []
+    this.score = 0
+    this.live = true
+  }
+
+  getRollCounter() {
+    return this.rollCounter;
+  }
+  getScore() {
+    return this.score;
+  }
+  isLive() {
+    return this.live;
+  }
+  addRoll() {
+    return this.rollCounter++;
+  }
+  addPinsToScore(pins) {
+    return this.score += pins;
+  }
```

Comment:
Similarly, as above, instead of storing a `score` variable, could you instead calculate the current score on-demand from the rolls array?